src/components/SearchCombobox.vue

```diff
@@ -134,17 +115,21 @@ function setMinIndex() {
 }
 
 function selectItem(focusNext = false) {
-  const item = workingList.value[activeIndex.value];
+  // setTimeout(() => {
+  showList.value = false;
+  // }, 100);
+
+  const item = props.listItems[activeIndex.value];
   if (item) {
     emit("update:modelValue", item.value);
   }
   if (focusNext && inputRef.value) {
     focusNextFormItem(inputRef.value);
```

Comment:
Dit is niet nieuw in dit PR, maar: als je in het contactverzoek formulier begint met een groep, is het medewerker veld disabled. Zodra je een groep kiest met de enter toets, wordt het medewerker veld niet snel genoeg enabled, waardoor de focus doorspringt naar een veld verder. Dit kunnen we oplossen in de focusNextFormItem functie door de setTimeout die daarin gebruikt wordt, eerder in de functie te trekken.

Reply:
wat bedoel je met "eerder in de functie te trekken"?
bedoel je zoiets?

 ```
setTimeout(() => {
  const elements = Array.from(
    element.form.querySelectorAll(
      'button:not([disabled]), [href], input:not([disabled]), 
    select:not([disabled]), textarea:not([disabled]), 
    [tabindex]:not([tabindex="-1"]):not([disabled]), 
    details:not([disabled]), summary:not(:disabled)'
      )
    );
    const i = elements.indexOf(element);
    if (i === -1 || i >= elements.length) return;
    const next = elements[i + 1];
    if (next instanceof HTMLElement) {
        next.focus();
      });
    }
 });
```
in plaats van 

```
  const elements = Array.from(
    element.form.querySelectorAll(
      'button:not([disabled]), [href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), [tabindex]:not([tabindex="-1"]):not([disabled]), details:not([disabled]), summary:not(:disabled)'
    )
  );
  const i = elements.indexOf(element);
  if (i === -1 || i >= elements.length) return;
  const next = elements[i + 1];
  if (next instanceof HTMLElement) {
    setTimeout(() => {
      next.focus();
    });
  }
```

Reply:
Ja precies!